### PR TITLE
🐛 Fixing various issues with the Storage Explorer

### PR DESCRIPTION
**Pull request by Sean-Stilwell**
# Pull Request

## Description

This PR fixes a variety of issues with the Storage Explorer, focusing on i18n.

## Related Issues

[Bug 8494](https://dev.azure.com/DataSolutionsDonnees/FSDH%20SSC/_workitems/edit/8494)
[Bug 8495](https://dev.azure.com/DataSolutionsDonnees/FSDH%20SSC/_workitems/edit/8495)
[Bug 8496](https://dev.azure.com/DataSolutionsDonnees/FSDH%20SSC/_workitems/edit/8496)
[Bug 8503](https://dev.azure.com/DataSolutionsDonnees/FSDH%20SSC/_workitems/edit/8503)
[Bug 8533](https://dev.azure.com/DataSolutionsDonnees/FSDH%20SSC/_workitems/edit/8533)

## Changes

- Adds missing translations for
    - Renaming file
    - Deleting file
    - Creating folder
    - Deleting folder
- Corrects behaviour of delete folder to display the open folder's name instead of "/"
- Removes "root" when displaying current directory.
- Fixes a variety of translations in the "Properties" section.
- Replaces AeButton with DHButton for the actions.

## Testing

- Locally tested

## Checklist

- [x] Code follows dotnet coding standards
- [x] Tests added/updated to cover changes

### CODE REVIEW

File: Portal/src/Datahub.Portal/i18n/localization.fr.json [hunk 1]
```diff
@@ -227,10 +227,10 @@
   "ArcGIS": "ArcGIS",
   "ArcGIS is a geographic information system (GIS) for working with maps and geographic information. It is used for creating and using maps, compiling geographic data, analyzing mapped information, sharing and discovering geographic information, using maps and geographic information in a range of applications, and managing geographic information in a database.": "ArcGIS est un système d'information géographique (SIG) pour travailler avec des cartes et des informations géographiques. Il est utilisé pour créer et utiliser des cartes, compiler des données géographiques, analyser des informations cartographiées, découvrir des informations géographiques dans une gamme d'applications.",
   "Architecture": "Architecture",
-  "Are you sure you want to delete file \\": "Etes-vous sûr de vouloir supprimer le fichier \\",
-  "Are you sure you want to delete folder \\": "Etes-vous sûr de vouloir supprimer le dossier \\",
+  "Are you sure you want to delete file \"{0}\"?": "Êtes-vous sûr de vouloir supprimer le fichier \"{0}\"?",
```
Comment (ErikApption) [inline Portal/src/Datahub.Portal/i18n/localization.fr.json:230]: technically there should be a space before the question mark

File: Portal/src/Datahub.Portal/i18n/localization.fr.json [hunk 2]
```diff
@@ -227,10 +227,10 @@
...
+  "Are you sure you want to delete folder \"{0}\"?": "Êtes-vous sûr de vouloir supprimer le dossier \"{0}\"?",
```
Comment (ErikApption) [inline Portal/src/Datahub.Portal/i18n/localization.fr.json:231]: same

File: Portal/src/Datahub.Portal/i18n/localization.fr.json [hunk 3]
```diff
@@ -227,10 +227,10 @@
...
   "Are you sure you want to delete the following project?": "Êtes-vous sûr de vouloir supprimer le projet suivant?",
-  "Are you sure you want to delete the following workspace: {0}?": "Êtes-vous sûr de vouloir supprimer l'espace de travail suivant : {0}?",
+  "Are you sure you want to delete the following workspace: {0}?": "Êtes-vous sûr de vouloir supprimer l'espace de travail suivant: {0}?",
```
Comment (ErikApption) [inline Portal/src/Datahub.Portal/i18n/localization.fr.json:233]: same

**Review by ErikApption — CHANGES_REQUESTED** [review]

File: Portal/src/Datahub.Portal/i18n/localization.fr.json (hunk 1 above)
Comment (Sean-Stilwell) [reply Portal/src/Datahub.Portal/i18n/localization.fr.json:230]: In Canadian French there is no space, DeepL likes to play this trick on us.

[Article from the Office québécois de la langue française](https://vitrinelinguistique.oqlf.gouv.qc.ca/22039/la-typographie/espacement/espacement-avant-et-apres-les-signes-de-ponctuation-et-les-symboles)

Comment (ErikApption) [reply Portal/src/Datahub.Portal/i18n/localization.fr.json:230]: Thank you - I checked it in word, and it defaulted to French / France, missed this nuance.

**Review by ErikApption — APPROVED** [review]

**Pull request merged**